pkgs/by-name/wi/within/package.nix

```
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "within";
+  version = "1.1.4";
+
+  src = fetchFromGitHub {
+    owner = "sjmulder";
+    repo = "within";
+    tag = finalAttrs.version;
+    hash = "sha256-UyOgEe07K1LW5IbB7ngxelp+9Njq/NPPkWw3sxAQyVY=";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
```

I wonder how this would pass.

Could @examosa or some GNU Make expert teach me how Make would treat `make PREFIX=\$(out)`? I'm surprised that it passes the build.

If this pattern works, how about the following?

```suggestion
  makeFlags = [ "PREFIX=$(prefix)" ];
```

Since `prefix` is known by `pkgs/stdenv/generic/setup.sh`:

https://github.com/NixOS/nixpkgs/blob/899c40e6f8ed0baee363a7a5d3be39241a40a1b7/pkgs/stdenv/generic/setup.sh#L960-L965